polaris-migrator/src/utilities/sass.ts

```diff
@@ -301,3 +325,16 @@ export function createSassMigrator(name: string, ruleFn: PolarisMigrator) {
     }).css;
   };
 }
+
+export function createSassMigrator(name: string, ruleFn: PolarisMigrator) {
+  const wrappedRule = ruleFn as StylelintRule;
```

Comment:
This gives us a logical point to inject further functionality (see the next PR where we also shim stylelint's `report()` method)